src/main/java/ch/njol/skript/classes/data/DefaultComparators.java

```diff
@@ -653,6 +654,14 @@ public boolean supportsOrdering() {
 				return false;
 			}
 		});
+
+		// Potion Effect Type
+		Comparators.registerComparator(PotionEffectType.class, PotionEffectType.class, new Comparator<PotionEffectType, PotionEffectType>() {
+			@Override
+			public Relation compare(PotionEffectType effectOne, PotionEffectType effectTwo) {
+				return Relation.get(effectOne.equals(effectTwo));
+			}
+		});
```

Comment:
I think you can register this like
```suggestion
		Comparators.registerComparator(PotionEffectType.class, PotionEffectType.class, (one, two) -> Relation.get(one.equals(two)));
```

Reply:
done and added, thanks